modules/structs/repo_commit.go

```diff
@@ -45,6 +45,7 @@ type Commit struct {
 	Author     *User         `json:"author"`
 	Committer  *User         `json:"committer"`
 	Parents    []*CommitMeta `json:"parents"`
+	Files      []string      `json:"files"`
```

Comment:
this is different to GH api responce so either rename it ... or make it similar

- sorry

Reply:
Hi there, sorry for the delay.
I'll fix the errors mentioned above.
Also, I'll look back at how github handle things : I based my code upon our existing handling of github api, might be outdated. 
Thanks for the review anyway.
Laurent